.github/workflows/chromatic.yml

```diff
@@ -0,0 +1,19 @@
+name: "Chromatic"
+on: push
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+    - name: Use Node.js
+      uses: actions/setup-node@v1
+      with:
+        node-version: '10.16.3'
+    - name: Install dependencies
+      run: yarn setup
+    - uses: chromaui/action@v1
+      with:
+        appCode: azwytw4amgi
```

Comment:
I intended to use `${{ secrets.CHROMATIC_APP_CODE }}` and put the value in the Github settings for repo, but I don't seem to have the right level of access to do that. The code is for a free trial account. We'd creata new account and/or change the key before going to production, and also delete this account/"chromatic project".

Reply:
@didoesdigital or @ActuallyACat would be able to set up the secret.